static/js/app.js

```diff
@@ -53,12 +53,12 @@ async function updateEmailForwardingPrefs(submitEvent) {
       if (prefToggle.value === "Enable") {
         prefToggle.title = "Disable email forwarding for this alias";
         toggleLabel.textContent = "forwarding";
-        wrappingEmailCard.classList.add("card-enabled");
+        wrappingEmailCard.classList.add("is-enabled");
         return prefToggle.value = "Disable";
       } else if (prefToggle.value === "Disable") {
         prefToggle.title="Enable email forwarding to this alias";
```

Comment:
Should these be localized?

Reply:
Yes. Filing this as a seperate issue.

Reply:
Filed! https://github.com/mozilla/fx-private-relay/issues/906